db/data_migration/20211129114611_move_phe_contacts_to_ukhsa.rb

```diff
@@ -0,0 +1,7 @@
+phe = Organisation.find_by!(slug: "public-health-england")
+ukhsa = Organisation.find_by!(slug: "uk-health-security-agency")
+
+phe.contacts.update_all(contactable_id: ukhsa.id)
+
+Whitehall::PublishingApi.republish_async(phe)
+Whitehall::PublishingApi.republish_async(ukhsa)
```

Comment:
Is there a reason you've done this as a database migration rather than as a rake task to be run once and then deleted?